01/app.js

```diff
@@ -0,0 +1,15 @@
+const init = function(btn) {
+    console.log("DOM został wczytany");
+
+};
+const btn = document.querySelector('button'); //?dev Jak to zrobić jakbym chhciał to umieścić w row 4?
```

Comment:
Normalnie, po prostu wstawić :P
Pewnie nie działa dlatego bo masz tam parametr o tej samej nazwie.

Reply:
hahahahah padłem - cyt.: "Normalnie, po prostu wstawić :P"
Zdecydowanie częściej muszę to przeglądać, bo się przynajmniej pośmiać można :D Zawsze to jakaś interakcja-społeczna  -> zuchwała do skocznia częściej frustrującej interakcji z samym komputerem :P